docs/start/framework/solid/environment-functions.md

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
once we have settled on the text for react, please duplicate and change the imports to point to solid

Reply:
@hokkyss see the docs in `docs/router/framework/solid/devtools.md` which have a syntax for performing text replacements on the `ref` doc. So you shouldn't need to duplicate the entire document.